### PR TITLE
Marketing fixes

### LINKED ISSUE #273: make new template for marketing pages with get-a-quote box on right side.
Screenshot 2024-10-03 at 2 23 41 AM

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #273, #272 , #270 , #271 

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us/marketing/shred-it-paper-shredding-services
  - https://main--stericycle-shared--aemsites.aem.page/en-us/service-locations/greenville-spartanburg-asheville - Regression test
- After: 
  - https://marketing-fixes--shredit--stericycle.aem.page/en-us/marketing/shred-it-paper-shredding-services
  - https://marketing-fixes--stericycle-shared--aemsites.aem.page/en-us/service-locations/greenville-spartanburg-asheville

### CODE REVIEW

**Comment by jindaliiita** [comment]
@MarquiseRosier These changes are breaking this page "**.column-with-list**" sections
https://marketing-fixes--shredit--stericycle.aem.live/en-us/secure-shredding-services/paper-shredding-services

File: styles/styles.css [hunk 1]
```diff
@@ -232,10 +232,16 @@ main li {
   list-style: none;
 }
 
+main .section.column-with-list.navy-background.columns-container {
```
Comment (jindaliiita) [inline styles/styles.css:235]: is these changes are related to the marketing pages, then ideally you should move this to the marketing template css

File: templates/marketing-location-page/marketing-location-page.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+div.page-sidebar > div.section > div.default-content-wrapper > h3 ~ p {
```
Comment (jindaliiita) [inline templates/marketing-location-page/marketing-location-page.css:1]: add **marketing-location-page** in the selector.

File: templates/service-location-page/service-location-page.css [hunk 3]
```diff
@@ -9,3 +9,32 @@ div.page-sidebar > div.section > div.default-content-wrapper > h3 ~ p:has(> a) {
 main .section > div.hero-wrapper {
     margin: unset;
 }
+
+.with-sidebar main > .body-wrapper {
```
Comment (jindaliiita) [inline templates/service-location-page/service-location-page.css:13]: use **service-location-page** in the selector

File: styles/styles.css [hunk 4]
```diff
@@ -551,6 +613,23 @@ main .section.with-image {
 
 /* section background image styles */
 
+main div.section.hard-drive-background {
+  background-size: contain;
+  background-position-x: 50%;
+  background-position-y: 100%;
+  background-image: url("/icons/media_18801dc9f2feef2abd2efd2bf6dfacd12e4469a20.png");
```
Comment (jindaliiita) [inline styles/styles.css:620]: I'm not sure if this is the right way to use a background image. Adding a URL like **'media_18801dc9f2feef2abd2efd2bf6dfacd12e4469a20.png'** is neither author-friendly nor developer-friendly.
Why we are not adding this in the document itself ?

File: styles/styles.css [hunk 5]
```diff
@@ -551,6 +613,23 @@ main .section.with-image {
...
+
+}
+
+main div.section.shredded-paper-background-column {
```
Comment (jindaliiita) [inline styles/styles.css:624]: Same comment as of above

File: styles/styles.css (hunk 1 above)
Comment (MarquiseRosier) [reply styles/styles.css:235]: these are changes to column-with-list!

File: templates/service-location-page-2/service-location-page-2.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
+.with-sidebar main > .body-wrapper {
```
[the remark this answers is not in the document]
Comment (MarquiseRosier) [reply templates/service-location-page-2/service-location-page-2.css:1]: this is resolved; this is code that was already in the codebase

**Comment by MarquiseRosier** [comment]
> @MarquiseRosier These changes are breaking this page "**.column-with-list**" sections https://marketing-fixes--shredit--stericycle.aem.live/en-us/secure-shredding-services/paper-shredding-services

should be fixed here @jindaliiita 

https://marketing-fixes--shredit--stericycle.aem.page/en-us/secure-shredding-services/paper-shredding-services

https://marketing-fixes--shredit--stericycle.aem.page/en-us/marketing/shred-it-pricing

**Review by dkuntze — APPROVED** [review]

**Pull request merged**